src/Stryker.CLI/README.md

```diff
@@ -67,6 +67,15 @@ If you want to decide on your own mutation score thresholds, you can configure t
 
 The defaults are respectivly: `80` `60` `0`
 
+#### Excluding files
+If you decide to exclude files for unit testing, you can configure this with the following command:
+
+`dotnet stryker --files-to-exclude ['./ExampleClass.cs','./ExampleDirectory/ExampleClass2.cs']`
+
+It is advisable to use relative paths. When someone else is running Stryker for you unit test project, paths are automatically resolved. It is still possible to use absolute paths to files.
+
+The only format which is accepted is a JSON formatted string array.
+When you want to exclude multiple files (for example more then 5) it is better to use the stryker configuration file because it is more easier to handle multiple files.
```

Comment:
```suggestion
When you want to exclude multiple files (for example more then 5) it is advised to use the stryker configuration file because it is easier to handle multiple files.
```